app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -271,10 +274,18 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
             className="flex flex-col items-center w-full pt-8"
             data-testid="weekly-picks"
           >
-            <h1 className="pb-8 text-center text-[2rem] font-bold text-foreground">
-              Week {week} pick
-            </h1>
-
+            <Heading 
+              as='h1'
+              className='pb-8'
+              data-testid='week__week-number'
+            >{`Week ${week} pick`}
+            </Heading>
+            <Heading
+              as='h2'
+              className='pb-8 text-muted-foreground'
+              data-testid='week__entry-name'
+            >{entryName}
```

Comment:
Sorry if this is a dumb question, but is there a reason we're not just using `currentEntry.name` here instead of creating state for it? It never changes, right?

Reply:
Oh nope, you’re right. Turns out it was a dumb question!

I’m so used to a different data fetching paradigm that I forgot this is necessary sometimes.

Reply:
Mike does bring up a good point here. This is not part of this ticket, but because we're now tracking 2 pieces of data from the response object, we should refactor the state to store the pick history and the entry name. A good ticket to refactor some code into the backlog.